nucliadb_protos/nodewriter.proto

```diff
@@ -77,8 +77,12 @@ message Counter {
     uint64 resources = 1;
 }
 
+message ShadowShardResponse {
+    bool success = 1;
+}
 
 service NodeSidecar {
-
     rpc GetCount(noderesources.ShardId) returns (Counter) {}
+    rpc ShadowShardCreate(noderesources.ShardId) returns (ShadowShardResponse) {}
```

Comment:
Would it be better if the parameter would be an `EmptyQuery` and we would let the sidecar itself create the shadow shard with a random id?